src/summary_writer.jl

```diff
@@ -4,7 +4,7 @@ import ..TensorFlow
 const tf = TensorFlow
 import ..TensorFlow: tensorflow, Graph, get_def_graph, @py_proc
 
-struct FileWriter
+struct FileWriter <: tf.Context
```

Comment:
:mindblown:

Reply:
Ya, we'll see if this is a good idea. It holds up for now though.